index.html

```diff
@@ -204,5 +204,19 @@ <h2 class="title">About Myself</h2>
             </ul>
             </div>
         </section>
+
+        <section class="contact">
+          <header>
+            <h2> Contact me </h2>
+            <p> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it  </p>
+          </header>
```

Comment:
- [ ] :thinking:  I think it is not necessary to wrap the title and description into a header, kindly remove it :recycle: